src/agent/src/main.rs

```diff
@@ -384,10 +384,20 @@ async fn start_sandbox(
     image::set_proxy_env_vars().await;
 
     #[cfg(feature = "agent-policy")]
-    if let Err(e) = initialize_policy().await {
-        error!(logger, "Failed to initialize agent policy: {:?}", e);
-        // Continuing execution without a security policy could be dangerous.
-        std::process::abort();
+    {
+        let mut policy_file = config.policy_file.as_str();
+        if policy_file.is_empty() {
+            warn!(
```

Comment:
If I understand correctly, users will see this warning if they don't configure anything, and they are using a Kata CI image. So, we should replace this warning with something like:

info!("Using default policy from {policy_file}");

Reply:
Yes makes sense. Will correct it.